Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -172,6 +172,8 @@ Configure the network that will be enabled for the cluster and services.
           **IMPORTANT**: Encryption requires the 5.11 kernel for the latest nbd and cephfs drivers. Alternatively for testing only,
           set "mounter: rbd-nbd" in the rbd storage class, or "mounter: fuse" in the cephfs storage class.
           The nbd and fuse drivers are *not* recommended in production since restarting the csi driver pod will disconnect the volumes.
+          If kernel version matches the above requirement, for CephFS set kernel mount options to ms_mode=secure,
```

Comment:
How about if we move this comment above the kernel requirement (~line 171), and mention which setting?
```suggestion
          If this setting is enabled, CephFS volumes also require setting `CSI_CEPHFS_KERNEL_MOUNT_OPTIONS` to `"ms_mode=secure"` in operator.yaml.
```

Reply:
Thanks for the suggestion :+1: